File: PeopleSearchUI/ClientApp/src/app/people-search/person.ts
export class Person {
  public id: number;
  public firstName: string;
  public lastName: string;
  public address: string;
  public age: number;
  public interests: string;
  simage: any[];

  get image() {
    return 'data:image/jpeg;base64, ' + this.simage;
  }

  constructor(data?: any) {
    Object.assign(this, assignData(data));
  }
}

function assignData(data) {
  return {
    id: data.id,
    firstName: data.firstName,
    lastName: data.lastName,
    address: data.address,
    age: data.age,
    interests: data.interests,
    simage: data.image
  }
}

File: PeopleSearchData/Models/Person.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Text;

namespace PeopleSearchData.Models
{
    public class Person
    {
        [Key]
        public int Id { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string Address { get; set; }
        public int Age { get; set; }
        public string Interests { get; set; }
        public byte[] Image { get; set; }
    }
}
